### PR TITLE
[Spotify Player] Create new queue command

### LINKED ISSUE #9704: [Spotify Player] Ability to see the Queue
### Extension

https://www.raycast.com/mattisssa/spotify-player

### Description

The ability to see the queue and skip to a song or rearrange it from Raycast would be pretty cool.

### Who will benefit from this feature?

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by bkeys818**
## Description

Adds a new command that allows the user to view their queue.

Closes #9704

## Screencast

Raycast extension Spotify Player - New queue command

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/spotify-player/package.json [hunk 1]
```diff
@@ -324,6 +324,13 @@
           "required": true
         }
       ]
+    },
+    {
+      "name": "queue",
+      "title": "Queue",
+      "subtitle": "Spotify",
+      "description": "See your queue and remove any tracks or episodes from it.",
+      "mode": "view"
```
Comment (peduarte) [inline extensions/spotify-player/package.json:333]: Can you add `"default": false` to this please

**Comment by peduarte** [comment]
I think it's cool to see the queue, but I was surprised when I couldn't play a specific item from the queue. Why is that? Only seeing the queue doesn't seem to add that much value imo.

**Comment by bkeys818** [comment]
Yeah, I'm not sure what I was thinking. I used `EpisodeListItem` and `TrackListItem`, so the queue items have actions now.

**Comment by peduarte** [comment]
Much cleaner ✨

File: extensions/spotify-player/package.json [hunk 2]
```diff
@@ -324,6 +324,14 @@
           "required": true
         }
       ]
+    },
+    {
+      "name": "queue",
+      "title": "Queue",
+      "subtitle": "Spotify",
+      "description": "See your queue and remove any tracks or episodes from it.",
```
Comment (peduarte) [inline extensions/spotify-player/package.json:332]: Is this description accurate? Can you remove tracks from it?

**Comment by mattisssa** [comment]
Such a cool command 👏
Wanted to build one for loooong 😬

File: extensions/spotify-player/package.json (hunk 2 above)
Comment (bkeys818) [reply extensions/spotify-player/package.json:332]: Oops, good cache

**Review by peduarte — APPROVED** [review]

**Pull request merged**